documents-guide.md

```diff
@@ -103,7 +103,25 @@ If you are making a document that needs a version label, use this:
 ### Version 1.0 {.version}
```

Comment:
Might be worth bumping the version?

Reply:
This is not the actual version of this document, it's an example of how you can use the version class. This doc doesn't have versioning, we can just update it like the README.